https://patchwork.ozlabs.org/patch/555979/
From 60f841f8011057d65b184bc7639503c2599ea502 Mon Sep 17 00:00:00 2001
From: "matwey.kornilov@gmail.com" <matwey.kornilov@gmail.com>
Date: Sat, 12 Dec 2015 12:27:08 +0300
Subject: [PATCH] am335x_evm: Don't undef CONFIG_BOOTDELAY

config_distro_defaults.h is already included in configs/ti_am335x_common.h (by means of ti_armv7_common.h)
We don't need to #undef CONFIG_BOOTDELAY because it drops default bootdelay which already defined on this line.

Signed-off-by: Matwey V. Kornilov <matwey.kornilov@gmail.com>
---
 include/configs/am335x_evm.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index c51db8c..cf6a606 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -18,10 +18,6 @@
 
 #include <configs/ti_am335x_common.h>
 
-/* Don't override the distro default bootdelay */
-#undef CONFIG_BOOTDELAY
-#include <config_distro_defaults.h>
-
 #ifndef CONFIG_SPL_BUILD
 #ifndef CONFIG_FIT
 # define CONFIG_FIT
-- 
2.1.4

